Documentation/en-us/TestUsingTestDoubles.md

```diff
@@ -0,0 +1,131 @@
+# Testing with Mock
+
+## Test doubles
+
+Following problem comes up often when writing tests. Here, `B` Class uses(depends on) `A Class`.
+
+![](https://github.com/Quick/Assets/blob/master/Screenshots/TestUsingMock_BusesA.png)
+
+Here's one example: BTests tests B, which uses A. Now bugs in A cause BTests to fail, even though B is fine! 
+All the failures are confusing--it's hard to answer the question "what's broken?"
+
+To avoid this problem, we can use a stand-in for A in Btests. In this case, we call stand-in for A `A' class`.
```

Comment:
I find `A` and `A'` hard to distinguish visually. Can we name the "stand-in for `A`" something else? Maybe `StandInA`?

In general, I think I prefer using more fun examples. 😅  Rather than `class A` and `class A'`, these could be `class Airplane` and `class PaperAirplane`, or something. 😛